packages/filebrowser-extension/src/index.ts

```diff
@@ -1174,6 +1176,42 @@ function addCommands(
     selector: '.jp-DirListing-header',
     rank: 14
   });
+
+  app.commands.addKeyBinding({
+    command: CommandIDs.open,
+    selector: selectorBrowser,
+    keys: ['Enter']
```

Comment:
Wondering whether this `Enter` shortcut has some side-effects on the existing behavior of being able to create a new directory, naming it, and pressing enter to go into it?

Reply:
For example the behavior looks a bit off when testing on the Binder for this branch:

https://user-images.githubusercontent.com/591645/117675726-1da8ea80-b1ad-11eb-839d-012d9da349ea.mp4